tools/pipelines/build-api-markdown-documenter.yml

```diff
@@ -57,17 +57,25 @@ pr:
     - .prettierignore
     - tools/api-markdown-documenter
     - tools/pipelines/build-api-markdown-documenter.yml
-    - tools/pipelines/templates/build-npm-package.yml
+    - tools/pipelines/templates/1ES/build-npm-package.yml
     - tools/pipelines/templates/include-set-package-version.yml
     - tools/pipelines/templates/include-vars.yml
     - tools/pipelines/templates/include-install-pnpm.yml
     - tools/pipelines/templates/include-use-node-version.yml
     - tools/pipelines/templates/include-process-test-results.yml
     - scripts/*
 
+variables:
+  - template: /tools/pipelines/templates/include-vars.yml@self
+    parameters:
+      publishOverride: '${{ parameters.publishOverride }}'
+      releaseBuildOverride: '${{ parameters.releaseBuildOverride }}'
+      buildNumberInPatch: ''
```

Comment:
Sorry, got to this a bit late. For future PRs, when we have empty arguments like this, I would recommend adding a quick comment explaining what it is / why we don't want it or want the default.